Dockerfile

```diff
@@ -22,17 +22,14 @@ LABEL vendor=Sonatype \
 
 ARG NEXUS_VERSION=3.5.2-01
```

Comment:
Let's standardize variable case between this and https://github.com/sonatype/docker-nexus-iq-server

Reply:
Kept this case to maintain compatibility with previous version of the Docker file.